experimental/datasets/hugging_face_dataset.py

```diff
@@ -125,6 +125,11 @@ def _load(self) -> Dict[str, CSVClassificationDataset]:
         return flair_datasets
 
     def to_flair_column_corpus(self) -> Corpus:
+        if not self.flair_datasets["train"]:
+            raise ValueError(
+                f"Hugging Face dataset {self.dataset_name} does not have " f"TRAIN subset."
+            )
+
         return Corpus(
             train=self.flair_datasets["train"],
             dev=self.flair_datasets["validation"],
```

Comment:
```suggestion
            dev=self.flair_datasets.get("validation"]),
```

Reply:
Apply same to test,  as I can't make suggestion to the line 136

Reply:
what is the purpose of this suggestion?

Reply:
There could be no validation or test key and it will return KeyError.  flair.data.Corpus allows empty dev or test split.